pkg/gardenlet/controller/shoot/shoot_control_reconcile.go

```diff
@@ -99,16 +100,25 @@ func (r *shootReconciler) runReconcileShootFlow(ctx context.Context, o *operatio
 		kubeProxyEnabled                = gardencorev1beta1helper.KubeProxyEnabled(o.Shoot.GetInfo().Spec.Kubernetes.KubeProxy)
 		shootControlPlaneLoggingEnabled = botanist.Shoot.IsShootControlPlaneLoggingEnabled(botanist.Config)
 		deployKubeAPIServerTaskTimeout  = defaultTimeout
+		deployEtcdTaskTimeout           = defaultTimeout
+		deployEtcdTaskInterval          = defaultInterval
```

Comment:
The `defaultInterval` is `5s` if I'm not mistaken and the deployment happens very often during CA rotation because the rollout takes a considerable amount of time. The motivation was just to save that many reconcile operations in Etcd-Druid. Do you still favor `5s` and want me to revert it?

Reply:
I think I haven't fully understood yet. What do you mean with "and the deployment happens very often during CA rotation because the rollout takes a considerable amount of time"? This interval should be used only when the task function fails and gets retried.

Reply:
Okay, got it. `DefaultInterval` is the wait period after the next retry. I thought that the operation `ctx` is restricted to the time of the `DefaultInterval`, my bad.